docs/UserGuide.md

```diff
@@ -226,7 +231,9 @@ If you want to override the code authorships deduced by RepoSense (which is base
 
 There are 2 types of `@@author` tags:
 - Start Tags (format: `@@author AUTHOR_GITHUB_ID`): A start tag indicates the start of a code segment written by the author identified by the `AUTHOR_GITHUB_ID`.
-- End Tags (format: `@@author`): Optional. If not provided, the code till the next start tag (or the end of the file) will be attributed to the author specified in the start tag above. Use only when necessary to minimize polluting your code with these extra tags.
+- End Tags (format: `@@author`): Optional. An end tag indicates the end of a code segment written by the author identified by the `AUTHOR_GITHUB_ID` of the start tag.
+
+> Note: If an end tag is not provided, the code till the next start tag (or the end of the file) will be attributed to the author specified in the start tag above. Use only when necessary to minimize polluting your code with these extra tags.
 
 The `@@author` tags should be enclosed within a comment, using the comment syntax of the file in concern. Below are some examples:
```

Comment:
In line `322`, the default file extensions should be all file formats within the repository.